app/controllers/posts_controller.rb

```diff
@@ -1,5 +1,5 @@
 class PostsController < ApplicationController
-  before_action :authenticate_user!
+  # load_and_authorize_resource
```

Comment:
- [ ] Post authorization should be handled by Cancan. KIndly uncomment this line to meet this requirement.